src/main/java/com/example/dto/CreateBookRequestDto.java

```diff
@@ -9,9 +13,15 @@
 @Setter
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateBookRequestDto {
+    @NotBlank
     private String title;
+    @NotNull
     private String author;
+    @NotNull
```

Comment:
```suggestion
    @NotBlank
```

Reply:
In this case I thought that it don't need change because it have @Isbn annotation.Have I correctli understood that if we have String field we always need write @NotBlank annotation?

Reply:
it depends :)